packages/api/utils/sqs.ts

```diff
@@ -0,0 +1,5 @@
+import { SQSClient } from "@aws-sdk/client-sqs";
+
+// Create SQS service object.
+const sqsClient = new SQSClient({});
+export { sqsClient };
```

Comment:
```suggestion
export const sqsClient = new SQSClient({});
```

Any reason not to do this?